docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -35,9 +35,15 @@ The `SignIn` object's properties can be split into logical groups, with each gro
 
 ### `create()`
 
-Use this method to kick-off the sign in flow. It creates a `SignIn` object and stores the sign-in lifecycle state.
+Use this method to initiate the sign-in process. It creates a new `SignIn` object and stores the sign-in lifecycle state.
```

Comment:
```suggestion
Returns a new `SignIn` object based on the params you pass to it, and stores the sign-in lifecycle state. Use this method to initiate the sign-in process.
```

I think in references we should state what the function does before stating what to do with it. Also, what do we mean by "stores the sign-in lifecycle state"? Is it stored in the `SignIn` object that gets returned?

Reply:
yes, in the `status` param !